data/json/morale_types.json

```diff
@@ -329,6 +329,16 @@
     "type": "morale_type",
     "text": "Dug out a grave"
   },
+  {
+    "id": "morale_accomplishment",
+    "type": "morale_type",
+    "text": "Accomplishment"
```

Comment:
This seems like to be a way too generic name. Imagine we add several reasons for this morale bonus to shoot, and the player could see several "Accomplishment"s in his morale display. I believe it needs to be specified to what type of accomplishment caused this morale boost.

Reply:
It was intentional. We should use generic morale types and less of them instead of creating more of similar types.

Reply:
There used to be a stacking issue, but now it's just a matter of avoiding clutter in the UI, which is still a pretty big deal.  The player is notified specifically what the accomplishment was when they perform the action, but minutes or hours later, does it matter what it was, or that they're happy about it?